src/components/RegisterForm.tsx

```diff
@@ -23,21 +23,34 @@ const RegisterForm = () => {
     event.preventDefault();
     usersService
       .register(firstName, lastName, email, password)
-      .then(() => login(dispatch, email, password))
+      .then(() => {
+        enqueueSnackbar("Successfully created account.", {
+          variant: "success",
+        });
+        login(dispatch, email, password);
+      })
       .catch((err) =>
         enqueueSnackbar(err, {
           variant: "error",
         })
       );
   };
 
+  ValidatorForm.addValidationRule(
```

Comment:
could this be reused not to duplicate in every file?

Reply:
could this be replaced with `minStringLength` rule? 
I found list of implemented rules from here
https://www.npmjs.com/package/react-form-validator-core

Reply:
Done